providers/go.mod

```diff
@@ -3,21 +3,64 @@ module k8s.io/cloud-provider-gcp/providers
 go 1.17
 
 require (
-	cloud.google.com/go v0.54.0
```

Comment:
Head of K/K 1.23 is v0.81.0 (https://github.com/kubernetes/kubernetes/blob/release-1.23/go.mod#L144) Are we sure we want to diverge?